app/src/main/java/org/p2p/wallet/history/ui/token/adapter/HistoryAdapter.kt

```diff
@@ -107,6 +107,7 @@ class HistoryAdapter(
             is MoonpayTransactionItem -> TRANSACTION_MOONPAY_VIEW_TYPE
             is UserSendLinksItem -> TRANSACTION_USER_SEND_LINKS_VIEW_TYPE
             is HistoryItem.SwapBannerItem -> SWAP_BANNER_VIEW_TYPE
+            else -> -1
```

Comment:
magic number detected. There's probably already a constant ERROR_VIEW_TYPE available

Reply:
yeah I will fix it in next PR, it's in progress